react/components/SummaryBox.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { FC, ReactNode } from 'react'
+import { useCssHandles, applyModifiers } from 'vtex.css-handles'
+import { TranslateEstimate } from 'vtex.shipping-estimate-translator'
+
+const CSS_HANDLES = [
+  'summarySection',
+  'summaryBox',
+  'summaryTitle',
+  'summaryContent',
+  'summaryItems',
+  'summaryShipping',
+  'summaryAddress',
+]
+
+interface Props {
+  type: 'pickup' | 'delivery'
+  title: ReactNode | string
+  itemQuantity: ReactNode | string
+  parcelQuantity: ReactNode | string
+  shippingEstimate: unknown
+  shippingAddress: ReactNode | string
```

Comment:
`ReactNode` can already be a string be a `string`

```typescript
type ReactNode = ReactChild | ReactFragment | ReactPortal | boolean | null | undefined;
type ReactText = string | number;
type ReactChild = ReactElement | ReactText;
```

Reply:
💅